tools/prom-metrics-collector/metrics_collector.go

```diff
@@ -26,15 +26,6 @@ import (
 	dto "github.com/prometheus/client_model/go"
 )
 
-// excludedMetrics defines the metrics to ignore,
-// open issue:https://github.com/kubevirt/containerized-data-importer/issues/2773
-// Do not add metrics to this list!
-var excludedMetrics = map[string]struct{}{
```

Comment:
Please keep this var and update the text to say:
This should be used only for very rare cases where the naming conventions that are explained in the best practices should be ignored.

Reply:
Done